<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Modifier</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2d2d2d;
}

QGroupBox {
    border: 2px solid #404040;
    border-radius: 5px;
    margin-top: 10px;
    color: #ffffff;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px 0 5px;
}

QLineEdit, QTextEdit, QPlainTextEdit, QSpinBox, QComboBox {
    background-color: #1e1e1e;
    border: 1px solid #404040;
    border-radius: 3px;
    padding: 5px;
    color: #ffffff;
    selection-background-color: #404040;
}

QPushButton {
    background-color: #2d2d2d;
    border: 1px solid #404040;
    border-radius: 3px;
    padding: 5px 10px;
    color: #ffffff;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #404040;
}

QPushButton:pressed {
    background-color: #505050;
}

QCheckBox {
    color: #ffffff;
}

QLabel {
    color: #ffffff;
}

QProgressBar {
    border: 1px solid #404040;
    border-radius: 3px;
    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: #14ffec;
    border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Настройки файлов</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Маски файлов:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fileMaskEdit">
         <property name="toolTip">
          <string>Маски файлов через запятую (например: *.txt,*.bin)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Исходная папка:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="sourcePathEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="sourceBrowseBtn">
           <property name="text">
            <string>Обзор</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Целевая папка:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="targetPathEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="targetBrowseBtn">
           <property name="text">
            <string>Обзор</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="recursiveCheck">
         <property name="text">
          <string>Рекурсивный поиск в подпапках</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Настройки обработки</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Удалять исходные:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="deleteSourceCheck">
         <property name="text">
          <string>Да</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>При конфликте имен:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="conflictCombo">
         <item>
          <property name="text">
           <string>Перезаписать</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Добавить счетчик</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Режим работы:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="modeCombo">
         <item>
          <property name="text">
           <string>Разовый запуск</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По таймеру</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Интервал (сек):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="intervalSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3600</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>XOR ключ (8 байт):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="xorKeyEdit">
         <property name="maxLength">
          <number>16</number>
         </property>
         <property name="toolTip">
          <string>16 hex символов (8 байт), например: 0123456789ABCDEF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="processNowBtn">
        <property name="text">
         <string>Обработать сейчас</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #14ffec; color: #000000;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startStopBtn">
        <property name="text">
         <string>Старт</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2ed573; color: #000000; font-weight: bold;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Прогресс выполнения</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Готов к работе</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Лог операций</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="logText">
         <property name="maximumBlockCount">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>